native/.idea/codeStyles/Project.xml

```diff
@@ -32,4 +34,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Were the changes to the codeStyles intentional?

Reply:
Yeah it is. It should prevent the auto-formatting from putting every parameter on a separate line.